lib/Mojo/WebSocketProxy/Backend/ConsumerGroups.pm

```diff
@@ -285,17 +315,18 @@ sub _next_request_id {
     return ++$self->{request_seq_id};
 }
 
+sub _rpc_category_timeout {
+    my ($self, $category_name) = @_;
+
+    return $self->category_timeout_config->{$category_name} // $self->timeout;
+}
+
 sub _send_request {
-    my ($self, $request_data) = @_;
+    my ($self, $request_data, $category_name) = @_;
 
     my $f = $self->loop->new_future;
     $self->redis->_execute(
-        xadd => XADD => (
-            'general',
-            ('MAXLEN', '~', '100000'),
-            '*',
-            $request_data->@*
-        ),
+        xadd => XADD => ($category_name, ('MAXLEN', '~', '100000'), '*', $request_data->@*),
```

Comment:
This nested list `('MAXLEN', '~', '100000')` doesn't really do what you'd expect; Perl will flatten it into the outer list (along with the contents of `$request_data->@*`.)  We could however use `qw(MAXLEN ~ 100000)` so as to reduce the need for quotes.

```suggestion
        xadd => XADD => ($category_name, qw(MAXLEN ~ 100000), '*', $request_data->@*),
```

Also, since our minimum Perl for this public CPAN module is `5.014`, we probably should bump https://github.com/binary-com/perl-Mojo-WebSocketProxy/blob/71fbbcf23c7dd5d1097dd8b9ed55b40f1e168ed0/cpanfile#L14 to `5.024` use `postderef`, otherwise we can't use it; seems we've had other postderef usage in other parts here but our dependencies have not been updated to reflect this change.

Reply:
Addressed.